core/controllers/question_editor.py

```diff
@@ -30,13 +30,15 @@ class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_id):
+    def post(self, skill_ids):
         """Handles POST requests."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
-            raise self.PageNotFoundException(
-                'The skill with the given id doesn\'t exist.')
+        skill_ids = skill_ids.split(',')
```

Comment:
Do not redefine variables. Call the input arg something like comma_separated_skill_ids.

Reply:
Done.